### PR TITLE
refactor!: remove startport (ignite) dependency 

### PR DESCRIPTION
**Pull request by alexanderbez**
Fixes https://github.com/palomachain/paloma/issues/554

# Background

* Removes the starport dependency.
* Remove all traces of "starport" from comments and docs
* Bump to SDK v0.45.10
* Remove third_party_proto
* Added a Makefile for building and proto targets

Note, most of the changes are superficial, due to proto generation and comment updates.

# Testing completed

* Correctness should be indicated by CI passing, including the build and tests passing, along with a general liveness test.

# Breaking changes

### CODE REVIEW

**Comment by measure-fi** [comment]
CI error:
```
Error: cmd/palomad/main.go:13:2: no required module provides package github.com/tendermint/starport/starport/pkg/cosmoscmd; to add it:
	go get github.com/tendermint/starport/starport/pkg/cosmoscmd
Error: Process completed with exit code 1.
```
(which might just mean go.sum needs updating)

File: proto/consensus/query.proto [hunk 1]
```diff
@@ -53,47 +56,43 @@ message QueryQueuedMessagesForSigningResponse {
...
+      [(gogoproto.casttype) = "github.com/cosmos/cosmos-sdk/types.ValAddress"];
 }
 
 message QueryMessagesInQueueResponse {
   repeated MessageWithSignatures messages = 1;
 }
 
-message QueryGetAllQueueNamesRequest {
-}
+message QueryGetAllQueueNamesRequest {}
```
Comment (measure-fi) [inline proto/consensus/query.proto:94]: There are a lot of gofmt changes in here.

**Comment by measure-fi** [comment]
@alexanderbez `scripts/protocgen.sh`

**Comment by alexanderbez** [comment]
```
➜ scripts/protocgen.sh
protoc-gen-gocosmos: program not found or is not executable
Please specify a program using absolute path or make sure the program is available in your PATH system variable
--gocosmos_out: protoc-gen-gocosmos: Plugin failed with status code 1.
evm/job.proto:4:1: warning: Import gogoproto/gogo.proto is unused.
evm/turnstone.proto:2:1: warning: Import gogoproto/gogo.proto is unused.
evm/query.proto:6:1: warning: Import cosmos/base/query/v1beta1/pagination.proto is unused.
evm/treasury.proto:2:1: warning: Import gogoproto/gogo.proto is unused.
protoc-gen-gocosmos: program not found or is not executable
Please specify a program using absolute path or make sure the program is available in your PATH system variable
--gocosmos_out: protoc-gen-gocosmos: Plugin failed with status code 1.
paloma/query.proto:6:1: warning: Import cosmos/base/query/v1beta1/pagination.proto is unused.
protoc-gen-gocosmos: program not found or is not executable
Please specify a program using absolute path or make sure the program is available in your PATH system variable
--gocosmos_out: protoc-gen-gocosmos: Plugin failed with status code 1.
scheduler/query.proto:6:1: warning: Import cosmos/base/query/v1beta1/pagination.proto is unused.
protoc-gen-gocosmos: program not found or is not executable
Please specify a program using absolute path or make sure the program is available in your PATH system variable
--gocosmos_out: protoc-gen-gocosmos: Plugin failed with status code 1.
treasury/query.proto:6:1: warning: Import cosmos/base/query/v1beta1/pagination.proto is unused.
protoc-gen-gocosmos: program not found or is not executable
Please specify a program using absolute path or make sure the program is available in your PATH system variable
--gocosmos_out: protoc-gen-gocosmos: Plugin failed with status code 1.
valset/snapshot.proto:5:1: warning: Import cosmos_proto/cosmos.proto is unused.
valset/snapshot.proto:7:1: warning: Import cosmos/base/v1beta1/coin.proto is unused.
valset/query.proto:6:1: warning: Import cosmos/base/query/v1beta1/pagination.proto is unused.
protoc-gen-gocosmos: program not found or is not executable
Please specify a program using absolute path or make sure the program is available in your PATH system variable
--gocosmos_out: protoc-gen-gocosmos: Plugin failed with status code 1.
cp: ./github.com/palomachain/paloma/*: No such file or directory
```

Typically, the target or script ensures all dependencies are fetched.

**Comment by taariq** [comment]
@alexanderbez yes. Let's push. We're interested.

**Comment by alexanderbez** [comment]
Ok, got most things working. Added a build target too, so I can test. The build is panicing, so I need to figure that out.

**Pull request merged**